crates/sui-types/src/multisig.rs

```diff
@@ -165,6 +176,11 @@ impl AuthenticatorTrait for MultiSig {
                         })?,
                     )
                 }
+                CompressedSignature::ZkLogin(s) => {
+                    // sender is consistent with multisig pk, do not verify author within zkLogin authenticator.
+                    s.verify_claims(value, author, verify_params, self.check_author())
```

Comment:
Do we ever call it with check_author() set to true? Wondering why not simplify by calling

`s.verify_claims(value, author, verify_params, false)`

Reply:
it means to retrieve the check_author() value from the enforced trait implementation for multisig instead of embedding in magic value. its safer to be enforced there